router/dataplane_concurrency_test.gobra

```diff
@@ -0,0 +1,491 @@
+package router
+
+import (
+	"sync"
+	io "verification/io"
+	sl "github.com/scionproto/scion/verification/utils/slices"
+)
+
+ghost
+decreases
+pure func ifsToIO_ifs(ifs uint16) option[io.IO_ifs]{
```

Comment:
Is this file meant to be merged eventually? If so, I will take a look at it as part of the pr. otherwise, we should drop it

Reply:
No, I will drop it.